Server/src/tests/loadTesting/signup-100.py

```diff
@@ -0,0 +1,17 @@
+# This python script signs up 100 different accounts
+
+import requests
+
+for lp in range(100):
+    account = "test" + str(lp) + "@123.com"
+    obj = {
+        'firstName': 'first',
+        'lastName': 'last',
+        'email': account, 
+        'dateofBirth': '2021-01-24 19:00:00',
+        'organizationName': 'Hogwarts',
+        'password': 1,
```

Comment:
I believe it would be better to use a longer password here, as we intend to eventually enforce a minimum length. Also as people themselves will want to use something better, even if it is the classic P@ssw0rd trick.

Reply:
True, in the UI the minimum length is 8. I modified the script and .csv to have "P@ssw0rd" as it's exactly 8